drivers/i2c/slave/CMakeLists.txt

```diff
@@ -1,3 +1,3 @@
 # SPDX-License-Identifier: Apache-2.0
 
-zephyr_sources_ifdef(CONFIG_I2C_EEPROM_SLAVE	eeprom_slave.c)
```

Comment:
Since the upper level `drivers/i2c` has a `zephyr_library()`, the object file will be included in I2C library. I could add `zephyr_library()` here if you want.

Reply:
ok, fair enough